tests/method-arg-name.rns [2.18]: first method argument must be called 'self'
    def __init__(i: int):
                 ^
